### PR TITLE
Re-enable CodeCov?

### PR DESCRIPTION
**Pull request by benjaoming**
I'm not what's the most elegant way to re-enable this, but it seems like another sequence of commands was originally running in this step

### CODE REVIEW

**Comment by benjaoming** [comment]
It has failed successfully :tada: :upside_down_face: 

```
Error: No coverage report found
Tip: See all example repositories: https://github.com/codecov?query=example

CircleCI received exit code 0

```

**Comment by oscarmcm** [comment]
I'm checking it and doing some subtle changes in the logic...

We already have the command in hatch https://github.com/django-wiki/django-wiki/blob/main/pyproject.toml#L106 and by default all of the commands in the default environment are replicated into the others this means that you can do something like `test:test` (use all the test matrix) or `test.py3.10-dj4.0:test` (use an specific test matrix)

**Comment by oscarmcm** [comment]
@benjaoming 

I'm having some troubles when trying to understand what this command does

```
	coverage run --source wiki setup.py test
```

can you explain a bit?

**Comment by benjaoming** [comment]
Hm, that must be a very old command?

In the "old days", the `coverage` tool would invoke the test framework and monitor source code access while the tests executed, so you'd write `coverage run <whatever external command>`

**Comment by benjaoming** [comment]
I guess this is still how things work, but now `coverage` is invoked by a `pytest` extension.

**Comment by oscarmcm** [comment]
Ok good news is I got the `codecov` command working 

```
---------- coverage: platform darwin, python 3.10.8-final-0 ----------
Name                                                                                       Stmts   Miss Branch BrPart  Cover
----------------------------------------------------------------------------------------------------------------------------
src/wiki/__init__.py                                                                           1      0      0      0   100%
src/wiki/admin.py                                                                             64     12      2      0    79%
src/wiki/apps.py                                                                              15      0      0      0   100%
src/wiki/checks.py                                                                            48      7     18      0    83%
src/wiki/conf/__init__.py                                                                      0      0      0      0   100%
src/wiki/conf/settings.py                                                                     70      1     10      1    98%
src/wiki/core/__init__.py                                                                      0      0      0      0   100%
src/wiki/core/diff.py                                                                          6      0      2      0   100%
src/wiki/core/exceptions.py                                                                    4      0      0      0   100%
src/wiki/core/http.py                                                                         35      5     14      5    80%
src/wiki/core/markdown/__init__.py                                                            56      5     16      5    86%
src/wiki/core/markdown/mdx/__init__.py                                                         0      0      0      0   100%
src/wiki/core/markdown/mdx/codehilite.py                                                      66      2     12      3    94%
src/wiki/core/markdown/mdx/previewlinks.py                                                    15      2      6      1    76%
src/wiki/core/markdown/mdx/responsivetable.py                                                 30      0      6      0   100%
src/wiki/core/paginator.py                                                                    23      0      8      1    97%
src/wiki/core/permissions.py                                                                  62     21     46      9    61%
src/wiki/core/plugins/__init__.py                                                              0      0      0      0   100%
src/wiki/core/plugins/base.py                                                                 25      2      0      0    92%
src/wiki/core/plugins/loader.py                                                                3      0      0      0   100%
src/wiki/core/plugins/registry.py                                                             42      1     10      2    94%
src/wiki/core/utils.py                                                                         3      0      0      0   100%
src/wiki/decorators.py                                                                        83     17     42     11    76%
src/wiki/editors/__init__.py                                                                  12      0      4      0   100%
src/wiki/editors/base.py                                                                      15      1      0      0    93%
src/wiki/editors/markitup.py                                                                  23      2      2      1    88%
src/wiki/forms.py                                                                            291     64     96     28    73%
src/wiki/forms_account_handling.py                                                            52      0     12      0   100%
src/wiki/managers.py                                                                         103     26     16      3    71%
src/wiki/migrations/0001_initial.py                                                           10      0      0      0   100%
src/wiki/migrations/0002_urlpath_moved_to.py                                                   6      0      0      0   100%
src/wiki/migrations/0003_mptt_upgrade.py                                                       6      0      0      0   100%
src/wiki/migrations/__init__.py                                                                0      0      0      0   100%
src/wiki/models/__init__.py                                                                   27      1      6      1    94%
src/wiki/models/article.py                                                                   236     25     60      6    84%
src/wiki/models/pluginbase.py                                                                133     50     30      2    55%
src/wiki/models/urlpath.py                                                                   191     30     58      8    80%
src/wiki/plugins/__init__.py                                                                   0      0      0      0   100%
src/wiki/plugins/attachments/__init__.py                                                       1      0      0      0   100%
src/wiki/plugins/attachments/admin.py                                                          9      0      0      0   100%
src/wiki/plugins/attachments/apps.py                                                           6      0      0      0   100%
src/wiki/plugins/attachments/forms.py                                                        110     43     22      6    58%
src/wiki/plugins/attachments/markdown_extensions.py                                           43      0     10      1    98%
src/wiki/plugins/attachments/migrations/0001_initial.py                                        9      0      0      0   100%
src/wiki/plugins/attachments/migrations/0002_auto_20151118_1816.py                             6      0      0      0   100%
src/wiki/plugins/attachments/migrations/__init__.py                                            0      0      0      0   100%
src/wiki/plugins/attachments/models.py                                                       120     15     35      8    84%
src/wiki/plugins/attachments/settings.py                                                      14      1      2      1    88%
src/wiki/plugins/attachments/urls.py                                                           3      0      0      0   100%
src/wiki/plugins/attachments/views.py                                                        235     91     46      9    55%
src/wiki/plugins/attachments/wiki_plugin.py                                                   19      0      2      0   100%
src/wiki/plugins/editsection/__init__.py                                                       1      0      0      0   100%
src/wiki/plugins/editsection/apps.py                                                           6      0      0      0   100%
src/wiki/plugins/editsection/markdown_extensions.py                                           85      7     30      5    88%
src/wiki/plugins/editsection/settings.py                                                       5      0      0      0   100%
src/wiki/plugins/editsection/views.py                                                        109      8     34      8    89%
src/wiki/plugins/editsection/wiki_plugin.py                                                   11      0      0      0   100%
src/wiki/plugins/globalhistory/__init__.py                                                     1      0      0      0   100%
src/wiki/plugins/globalhistory/apps.py                                                         6      0      0      0   100%
src/wiki/plugins/globalhistory/forms.py                                                        0      0      0      0   100%
src/wiki/plugins/globalhistory/models.py                                                       0      0      0      0   100%
src/wiki/plugins/globalhistory/settings.py                                                     1      0      0      0   100%
src/wiki/plugins/globalhistory/views.py                                                       23      0      2      0   100%
src/wiki/plugins/globalhistory/wiki_plugin.py                                                  9      0      0      0   100%
src/wiki/plugins/help/__init__.py                                                              1      1      0      0     0%
src/wiki/plugins/help/apps.py                                                                  6      6      0      0     0%
src/wiki/plugins/help/models.py                                                                0      0      0      0   100%
src/wiki/plugins/help/views.py                                                                 0      0      0      0   100%
src/wiki/plugins/help/wiki_plugin.py                                                           8      8      2      0     0%
src/wiki/plugins/images/__init__.py                                                            1      0      0      0   100%
src/wiki/plugins/images/admin.py                                                              22      6      2      0    67%
src/wiki/plugins/images/apps.py                                                               10      0      0      0   100%
src/wiki/plugins/images/checks.py                                                              9      1      2      1    82%
src/wiki/plugins/images/forms.py                                                              52      2      6      2    93%
src/wiki/plugins/images/markdown_extensions.py                                                38      0      2      0   100%
src/wiki/plugins/images/migrations/0001_initial.py                                             6      0      0      0   100%
src/wiki/plugins/images/migrations/0002_auto_20151118_1811.py                                  6      0      0      0   100%
src/wiki/plugins/images/migrations/__init__.py                                                 0      0      0      0   100%
src/wiki/plugins/images/models.py                                                             94     22     24      6    71%
src/wiki/plugins/images/settings.py                                                            9      0      0      0   100%
src/wiki/plugins/images/templatetags/__init__.py                                               0      0      0      0   100%
src/wiki/plugins/images/templatetags/wiki_images_tags.py                                      12      1      2      1    86%
src/wiki/plugins/images/templatetags/wiki_thumbnails.py                                        1      0      0      0   100%
src/wiki/plugins/images/views.py                                                             123     27     22      6    73%
src/wiki/plugins/images/wiki_plugin.py                                                        21      0      4      1    96%
src/wiki/plugins/links/__init__.py                                                             1      0      0      0   100%
src/wiki/plugins/links/apps.py                                                                 6      6      0      0     0%
src/wiki/plugins/links/mdx/__init__.py                                                         0      0      0      0   100%
src/wiki/plugins/links/mdx/djangowikilinks.py                                                 77     23     24      2    65%
src/wiki/plugins/links/mdx/urlize.py                                                          40      0      8      0   100%
src/wiki/plugins/links/models.py                                                               0      0      0      0   100%
src/wiki/plugins/links/settings.py                                                             2      2      0      0     0%
src/wiki/plugins/links/views.py                                                               16     16      4      0     0%
src/wiki/plugins/links/wiki_plugin.py                                                         16     16      2      0     0%
src/wiki/plugins/macros/__init__.py                                                            1      0      0      0   100%
src/wiki/plugins/macros/apps.py                                                                6      0      0      0   100%
src/wiki/plugins/macros/mdx/__init__.py                                                        0      0      0      0   100%
src/wiki/plugins/macros/mdx/macro.py                                                          46      9     12      5    76%
src/wiki/plugins/macros/mdx/toc.py                                                            22      0      4      1    96%
src/wiki/plugins/macros/mdx/wikilinks.py                                                      38      2      6      2    91%
src/wiki/plugins/macros/settings.py                                                            4      0      0      0   100%
src/wiki/plugins/macros/templatetags/__init__.py                                               0      0      0      0   100%
src/wiki/plugins/macros/templatetags/wiki_macro_tags.py                                       16      2      2      0    89%
src/wiki/plugins/macros/wiki_plugin.py                                                         9      0      2      1    91%
src/wiki/plugins/notifications/__init__.py                                                     1      0      0      0   100%
src/wiki/plugins/notifications/apps.py                                                        28      1     11      1    95%
src/wiki/plugins/notifications/forms.py                                                       93     21     24      7    71%
src/wiki/plugins/notifications/management/__init__.py                                          0      0      0      0   100%
src/wiki/plugins/notifications/management/commands/__init__.py                                 0      0      0      0   100%
src/wiki/plugins/notifications/management/commands/wiki_notifications_create_defaults.py      33     33     10      0     0%
src/wiki/plugins/notifications/migrations/0001_initial.py                                      5      0      0      0   100%
src/wiki/plugins/notifications/migrations/0002_auto_20151118_1811.py                           6      0      0      0   100%
src/wiki/plugins/notifications/migrations/__init__.py                                          0      0      0      0   100%
src/wiki/plugins/notifications/models.py                                                      35      3      8      1    91%
src/wiki/plugins/notifications/settings.py                                                     3      0      0      0   100%
src/wiki/plugins/notifications/util.py                                                         9      1      4      1    85%
src/wiki/plugins/notifications/views.py                                                       33      0      6      1    97%
src/wiki/plugins/notifications/wiki_plugin.py                                                 11      0      0      0   100%
src/wiki/plugins/redlinks/__init__.py                                                          1      0      0      0   100%
src/wiki/plugins/redlinks/apps.py                                                              6      0      0      0   100%
src/wiki/plugins/redlinks/mdx/__init__.py                                                      0      0      0      0   100%
src/wiki/plugins/redlinks/mdx/redlinks.py                                                     73      1     14      1    98%
src/wiki/plugins/redlinks/wiki_plugin.py                                                       5      0      0      0   100%
src/wiki/sites.py                                                                             82      1      6      1    98%
src/wiki/templatetags/__init__.py                                                              0      0      0      0   100%
src/wiki/templatetags/wiki_tags.py                                                           110      0     16      1    99%
src/wiki/urls.py                                                                              83     10     10      2    85%
src/wiki/views/__init__.py                                                                     0      0      0      0   100%
src/wiki/views/accounts.py                                                                   103     12     24     10    81%
src/wiki/views/article.py                                                                    604     97    150     32    79%
src/wiki/views/deleted_list.py                                                                17      1      6      1    91%
src/wiki/views/mixins.py                                                                      26      3      4      1    87%
----------------------------------------------------------------------------------------------------------------------------
TOTAL                                                                                       4674    773   1082    213    79%

=============================================================== 253 passed, 3 skipped, 222 warnings in 24.41s ================================================================
cmd [3] | codecov

      _____          _
     / ____|        | |
    | |     ___   __| | ___  ___ _____   __
    | |    / _ \ / _  |/ _ \/ __/ _ \ \ / /
    | |___| (_) | (_| |  __/ (_| (_) \ V /
     \_____\___/ \____|\___|\___\___/ \_/
                                    v2.1.12

==> Detecting CI provider
  -> Got branch from git/hg
  -> Got sha from git/hg
==> Preparing upload
==> Processing gcov (disable by -X gcov)
==> Collecting reports
    Generating coverage xml reports for Python
    + /Users/loki/Developer/django-wiki/coverage.xml bytes=226086
==> Uploading
    .url https://codecov.io
    .query commit=dca2808e8780428dd50b5b8f716dfa717196d459&branch=benjaoming-coverage-in-ci&package=py2.1.12
    Gzipping contents..
    Compressed contents to 17792 bytes
    Pinging Codecov...
Error: Could not determine repo and owner
Tip: See all example repositories: https://github.com/codecov?query=example
```

**Comment by oscarmcm** [comment]
But I still cannot make this command works https://github.com/django-wiki/django-wiki/blob/releases/0.9/Makefile#L59

It just keeps returning `Nothing to do.`

File: pyproject.toml [hunk 1]
```diff
@@ -86,8 +86,10 @@ dependencies = [
 ]
 
 [tool.hatch.envs.default.scripts]
-cov = "coverage --cov-config=pyproject.toml run --source wiki test && coverage report -m"
-no-cov = "cov --no-cov {args}"
+cov = [
```
Comment (oscarmcm) [inline pyproject.toml:89]: Still haven't found a way to make this works, but gonna pass it for now.

**Review by oscarmcm — APPROVED** [review]
LGTM 👍 

Thanks for addressing this!

**Comment by oscarmcm** [comment]
@benjaoming please feel free to merge this once you believe the coverage is working as expected.

File: pyproject.toml (hunk 1 above)
Comment (benjaoming) [inline pyproject.toml:89]:
```suggestion
# Ignore this entrypoint for now, as it's not fully implemented
cov = [
```

**Comment by benjaoming** [comment]
Nice work removing .coveragerc!

**Pull request merged**